### DISCUSSION TITLE
[M-Z] (9 Settembre 2019) Esercizio 4

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#450] [Esami & Esoneri M-Z]
### Traccia

<img width="942" alt="image" src="https://github.com/user-attachments/assets/bdeb5a36-cf78-4141-a61b-b0fa7fc32ea5">

**Comment by rimaout** [comment]
<img width="600" alt="image" src="https://github.com/user-attachments/assets/eed26882-a3b2-449e-98cd-b437af3bf5c3">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190909.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190909.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/68e840b3-f1c2-440d-8593-467ab2282b10)
